zkevm-circuits/src/pi_circuit.rs

```diff
@@ -1245,6 +1249,22 @@ impl<F: Field> SubCircuit<F> for PiCircuit<F> {
         let pi_cells = layouter.assign_region(
             || "region 0",
             |mut region| {
+                // Annotate columns
+
```

Comment:
Could you also call these so that the tables are also annotated within the region?
```rust
tx_table.annotate_columns_in_region(&mut region);
block_table.annotate_columns_in_region(&mut region);
```

This is necessary because the `tx_table.annotate_columns` before the region only give annotations for the lookups, but not for the gates used in the expressions.